jenkins/ring-install.sh

```diff
@@ -0,0 +1,331 @@
+#!/bin/bash -xue
+
+test -n "${SUP_ADMIN_LOGIN:-}" || (echo "SUP_ADMIN_LOGIN should be defined." && return 1);
+test -n "${SUP_ADMIN_PASS:-}" || (echo "SUP_ADMIN_PASS should be defined." && return 1);
+test -n "${INTERNAL_MGMT_LOGIN:-}" || (echo "INTERNAL_MGMT_LOGIN should be defined." && return 1);
+test -n "${INTERNAL_MGMT_PASS:-}" || (echo "INTERNAL_MGMT_PASS should be defined." && return 1);
+test -n "${DEBIAN_FRONTEND:-}" || (echo "DEBIAN_FRONTEND should be defined." && return 1);
```

Comment:
This variable is not needed because DEBIAN_FRONTEND must also be set to 'noninteractive'.

So this can remain as 'export DEBIAN_FRONTEND=noninteractive' ie the caller won't ever need to change this.